### PR TITLE
Add Guides on how to use Discord to engage with the Community

### PR DESCRIPTION
**Pull request by chrisgrieser**
## Added
- new Guide MoC: for Engaging with the Obsidian Community
- Guide on Using Discord

## Checklist

- [x] I have renamed all attached images with descriptive file names (or I did not include any images)
- [x] Before creating a new note, I searched the vault (or I only edited existing notes)
- [x] (Optional) In case I created a new note in the folder `04 - Guides, Workflows, & Courses`, I added a link to the new note(s) in one of the `for {group X}` overviews ([listed here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses)).

### CODE REVIEW

File: 04 - Guides, Workflows, & Courses/Guides/Using Discord.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+---
+
+# Using Discord
+
+## Don't ask to ask, just ask
```
Comment (argenos) [inline 04 - Guides, Workflows, & Courses/Guides/Using Discord.md:10]: A little introduction of how/why this is needed? Right off the bat these two things seem unrelated.

Comment (argenos) [reply 04 - Guides, Workflows, & Courses/Guides/Using Discord.md:10]: It might also be best at the end of the file, once we have explained some of the other Discord stuff?

**Review by argenos — CHANGES_REQUESTED** [review]

File: 04 - Guides, Workflows, & Courses/Guides/Using Discord.md (hunk 1 above)
Comment (chrisgrieser) [reply 04 - Guides, Workflows, & Courses/Guides/Using Discord.md]: good point, did both

**Review by argenos — APPROVED** [review]
